src/components/LoadingCensorBar.re

```diff
@@ -1,10 +1,10 @@
 module Styles = {
   open Css;
 
-  let main = (~w, ~h, ~r, ~colorBase, ~colorLighter, ()) =>
+  let main = (~w, ~h, ~r, ~colorBase, ~colorLighter, ~widthPercent, ()) =>
     style([
       display(`flex),
-      width(`px(w)),
+      width(widthPercent !== 0.0 ? `percent(widthPercent) : `px(w)),
```

Comment:
Why don't we use "0"?

Reply:
percent used as float number ka

Reply:
<img width="242" alt="Screen Shot 2565-02-08 at 16 51 23" src="https://user-images.githubusercontent.com/12908129/152962194-af19771f-8f5a-4eee-8676-037bf696811c.png">